apps/archive/commands.py

```diff
@@ -334,8 +339,11 @@ def _move_to_archived(self, item, filter_conditions):
                 logger.info('{} Found {} published items for item: {}'.format(self.log_msg,
                                                                               len(published_items), item_id))
                 if moved_to_archived:
-                    archived_service.post(published_items)
-                    logger.info('{} Moved item to text archive for item {}.'.format(self.log_msg, item_id))
+                    try:
+                        archived_service.post(published_items)
+                        logger.info('{} Moved item to text archive for item {}.'.format(self.log_msg, item_id))
+                    except Conflict:
+                        logger.info('{} Moved item already to text archive for item {}.'.format(self.log_msg, item_id))
```

Comment:
Not clear what does it mean, IMO, it's better to rephrase it.

Reply:
will remove catch after all